var/spack/repos/builtin/packages/py-ipython/package.py

```diff
@@ -26,10 +27,13 @@ class PyIpython(PythonPackage):
     depends_on('py-pygments',                   type=('build', 'run'))
     depends_on('py-pickleshare',                type=('build', 'run'))
     depends_on('py-simplegeneric@0.8:',         type=('build', 'run'))
-    depends_on('py-prompt-toolkit@1.0.4:1.999', when='@:7.0.0', type=('build', 'run'))
-    depends_on('py-prompt-toolkit@2.0.0:2.999', when='@7.0.0:', type=('build', 'run'))
+    depends_on('py-prompt-toolkit@1.0.4:1.999',  when='@:7.0.0', type=('build', 'run'))
+    depends_on('py-prompt-toolkit@2.0.0:2.999',  when='@7.0.0:', type=('build', 'run'))
+    depends_on('py-prompt-toolkit@2.0.0:2.0.999', when='@7.5.0:', type=('build', 'run'))
     depends_on('py-traitlets@4.2:',             type=('build', 'run'))
     depends_on('py-decorator',                  type=('build', 'run'))
     depends_on('py-pexpect',                    type=('build', 'run'))
     depends_on('py-backcall',                   type=('build', 'run'), when="^python@3.3:")
     depends_on('py-appnope', type=('build', 'run'), when='platform=darwin')
+    depends_on('py-jedi@0.10:', type=('build', 'run'), when='@7.5.0:')
+    depends_on('py-backcall', type=('build', 'run'), when='@7.5.0:')
```

Comment:
Missing dependencies:

* setuptools>=18.5 (possibly type=build/run)
* ':python_version == "3.4"': ['typing']

Reply:
7.x [require python_version >= 3.5](https://github.com/ipython/ipython/blob/7.5.0/setup.py#L29) , so this will never trigger

Reply:
setuptools - added